tests/test_Dataset.py

```diff
@@ -1267,6 +1267,34 @@ def _repeat2(input_iter: Iterator[TensorDict], **kwargs) -> Iterator[TensorDict]
     assert func(2) == 21
 
 
+def _repeat2(input_iter: Iterator[TensorDict], **kwargs) -> Iterator[TensorDict]:
+    for tdict in input_iter:
+        yield tdict
+        yield tdict
+
+
+def test_MultiProcPostprocessingDataset():
+    _demo_txt = "some utterance text that has a few words"
+    with create_ogg_zip_txt_only_dataset_opts(text=_demo_txt) as sub_ds_opts:
```

Comment:
This dataset only has a single seq, and then via your `_repeat2`, you get 2 seqs. I think for this test, better use some dataset with more seqs.

Actually, the test right now would not even really make use of the second worker at all? It would put the single seq to worker 1, and worker 1 would then call `map_seq_stream`, and worker 2 would do nothing.